packages/babel-plugin-check-es2015-constants/test/fixtures/general/no-for-in/expected.js

```diff
@@ -2,4 +2,5 @@ var MULTIPLIER = 5;
 
 for (MULTIPLIER in arr) {
   throw new Error("\"MULTIPLIER\" is read-only");
+  ;
```

Comment:
Always curious why we have these empty statements, is there a way to remove?

Reply:
I don't really know where this one's coming from.